sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-backend-v2/README.md

```diff
@@ -75,15 +75,15 @@ spring.security.oauth2.client.registration.azure.client-id=xxxxxx-your-client-id
 spring.security.oauth2.client.registration.azure.client-secret=xxxxxx-your-client-secret-xxxxxx
 spring.security.oauth2.client.registration.azure.client-name=Azure
 spring.security.oauth2.client.registration.azure.provider=azure-oauth-provider
-spring.security.oauth2.client.registration.azure.scope=openid, https://graph.microsoft.com/user.read
-spring.security.oauth2.client.registration.azure.redirect-uri-template={baseUrl}/login/oauth2/code/{registrationId}
+spring.security.oauth2.client.registration.azure.scope=openid, https://graph.microsoft.com/user.read, profile
+spring.security.oauth2.client.registration.azure.redirect-uri={baseUrl}/login/oauth2/code/{registrationId}
```

Comment:
We already used `azure-oauth-provider`, to keep same style, should we use `azure-client` instead of `azure` here?

Refs: https://docs.spring.io/spring-boot/docs/2.3.4.RELEASE/reference/html/spring-boot-features.html#boot-features-security-oauth2-client

Or rename all `azure-oauth-provider` to `azure`.

Reply:
I think both can make sense, let me replace `azure` to `azure-oauth-client`.